utils/build.ps1

```diff
@@ -3419,7 +3446,7 @@ if (-not $IsCrossCompiling) {
       "-TestLLVM" = $Test -contains "llvm";
       "-TestSwift" = $Test -contains "swift";
     }
-    Invoke-BuildStep Test-Compilers $HostPlatform $Tests
+    Invoke-BuildStep Test-Compilers $HostPlatform $PinnedToolchainVariant $Tests 
```

Comment:
We should always be testing the Asserts variant IMO.

Reply:
Fixed